lib/modules/dosomething/dosomething_reportback/dosomething_reportback.forms.inc

```diff
@@ -37,6 +37,15 @@ function dosomething_reportback_form($form, &$form_state, $entity = NULL) {
       '#required' => TRUE,
       '#default_value' => $entity->nid,
     );
+    $form['camapgin_run_nid'] = array(
+      '#type' => 'entity_autocomplete',
+      '#title' => t('Campaign Run ID'),
+      '#description' => t("The campaign run node this reportback was submitted for."),
+      '#entity_type' => 'node',
+      '#bundles' => array('campaign_run'),
+      '#required' => TRUE,
+      '#default_value' => $entity->nid,
```

Comment:
When hitting this form, both `Node` and `Campaign Run ID` fields are not populated with anything. Should I delete these lines because that will actually never work since we're creating a new entity and thus they shouldn't have a `#default_value`, right?